applications/HDF5Application/tests/test_HDF5Application.py

```diff
@@ -50,7 +51,7 @@ def run_cpp_unit_tests():
     # We set cwd in case the script is run from another directory. This is needed
     # when testing from the core.
     out_bytes = subprocess.check_output(
-        ['python3', 'run_cpp_unit_tests.py'], cwd=dirname(__file__))
+        ['python3', 'run_cpp_unit_tests.py'], cwd=abspath(dirname(__file__)))
```

Comment:
just for the record, with #5577 this should no longer be necessary

Reply:
@philbucher thanks for tip. I think it is still correct to change this here since e.g. `python3 /path-to-hdf5-tests/test_HDF5Application.py` should also work.